components/common-go/cgroups/v2/cpu.go

```diff
@@ -43,7 +43,7 @@ func (c *Cpu) Max() (quota uint64, period uint64, err error) {
 	path := filepath.Join(c.path, "cpu.max")
 	content, err := os.ReadFile(path)
 	if err != nil {
-		return 0, 0, nil
+		return 0, 0, err
```

Comment:
Could you build a preview environment for this PR, and confirm you can start workspaces? 

I wonder if max doesn't exist, briefly, when workspaces are created, until `ws-daemon` applies maximums on a rotational/regular/scheduled basis.

Reply:
The `cpu.max` file is automatically generated when the container is created. It is not generated by us. Perhaps for some reason the container was erased, and this code was executed after the cgroup was cleaned up.

Reply:
We can't manually update the `cpu.max` file, correct?
If yes, could we use a unit test to test it?

Reply:
@jenting Good point. we can turn off the cgroup directory by hand, but I think just covering it with unit test is a great idea.